controllers/release/adapter.go

```diff
@@ -373,6 +420,56 @@ func (a *adapter) createManagedPipelineRun(resources *loader.ProcessingResources
 	return pipelineRun, nil
 }
 
+// createTenantPipelineRun creates and returns a new tenant Release PipelineRun. The new PipelineRun will include owner
+// annotations, so it triggers Release reconciles whenever it changes. The Pipeline information and the parameters to it
+// will be extracted from the given ReleasePlan. The Release's Snapshot will also be passed to the release
+// PipelineRun.
+func (a *adapter) createTenantPipelineRun(releasePlan *v1alpha1.ReleasePlan, snapshot *applicationapiv1alpha1.Snapshot) (*tektonv1.PipelineRun, error) {
+	tenantPipelineParams := []tektonv1.Param{}
```

Comment:
I'd add this block to a `ToTektonParams` function in `pipeline.go`. I wouldn't block the PR for this. I'm ok with this being refactored later.

Reply:
I am pretty sure I was told to remove the previous function from there, that was doing exactly this.

Reply:
Ah, ok. The previous function was in the `pipeline_run_builder.go`. I will do that.

Reply:
Just don't make it tenant pipeline specific. have it just take a list of params and do the append

Reply:
new function added to `utils/pipeline.go`